docs/contributing/index.md

```diff
@@ -1,69 +1,93 @@
+
+!!! note
+Issues created for questions or unconfirmed bugs will be moved to GitHub discussions by the maintainers.
+
+## How can I contribute?
 
 ### Discussions
 
-You can contribute to Autoware by facilitating discussions:
+You can contribute to Autoware by facilitating and participating in discussions, such as:
+
+- [Proposing a new feature to enhance Autoware](https://github.com/orgs/autowarefoundation/discussions/categories/feature-requests).
+- [Joining an existing discussion and expressing your opinion](https://github.com/orgs/autowarefoundation/discussions).
+- Organizing discussions for other contributors.
+- [Answering questions and supporting other contributors](https://github.com/autowarefoundation/autoware/discussions/categories/q-a?discussions_q=category%3AQ%26A+is%3Aunanswered).
+
+### Working groups
+
+The [various working groups](https://github.com/autowarefoundation/autoware-projects/wiki#working-group-list) within the Autoware Foundation are responsible for managing low level activities as directed by the foundation's Technical Steering Committee. These working groups are open to everyone, and joining a particular working group will allow you to gain an understanding of current projects, see how those projects are managed within each group and contribute to issues that will help progress a particular project.
 
-- Find a bug and open a new issue.
-- Propose an idea of features that will enhance Autoware.
-- Join an existing discussion and express your opinion.
-- Organize discussions for other contributors.
-- Answer questions and support other contributors.
+To see the schedule for upcoming working group meetings, refer to the [Autoware Foundation events calendar](https://calendar.google.com/calendar/u/0/embed?src=autoware.org_6lol0ho5ft0217h8c60pi1fm30@group.calendar.google.com).
 
-See our [support guidelines](../help/support-guidelines.md) for the detailed steps.
+### Reporting bugs
+
+Before you submit an issue, please search the issue tracker for the appropriate repository. It is possible that someone has already reported the same issue and that workarounds exist. If you can't determine the appropriate repository, ask the maintainers for help by creating a new discussion in the [Q&A category](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
+
+In order to fix a bug, we need to reproduce and confirm it first and so you should provide a minimal reproduction. Having a minimal reproducible scenario allows us to quickly confirm a bug (or point out a coding or configuration problem) as well as confirm that we are fixing the right problem.
+
+If you want to fix the bug by yourself, then that will be appreciated, but you should discuss possible approaches with the maintainers first before submitting a pull request.
```

Comment:
I think this can be simplified as well. It would be even better if it has where contributors should make a discussion. (Q&A in GitHub discussions?) @kenji-miyake 

```
We appreciate if you fix a bug by yourself. Please discuss possible approaches with maintainers before sending a pull request
```

Reply:
@IshitaTakeshi @kenji-miyake My assumption was that the approach would be discussed in the issue itself, since this is where people will report confirmed and reproduceable bugs to the maintainers.

Reply:
How about this?

```suggestion
Providing a fix for your reported bug will be highly appreciated, but discuss possible approaches with the maintainers in the issue before submitting a pull request.
```

Reply:
LGTM
@kenji-miyake How about this?

Reply:
Considering this guideline, I prefer the original sentence because people want to fix a bug here.
https://developers.google.com/style/clause-order

Reply:
Done!